src/Aspire.Dashboard/Components/Controls/ResourceSelect.razor

```diff
@@ -8,6 +8,7 @@
               SelectedOptionChanged="SelectedResourceChanged"
               AriaLabel="@AriaLabel"
               Class="resource-list"
+              Position="SelectPosition.Below"
```

Comment:
Needed, as at some zoom/resolutions position can be above, and users are cut off from seeing the entire select options.